cypress/e2e/main.spec.js

```diff
@@ -148,9 +148,37 @@ context('Window', () => {
 
     it('should transition to pages using swup API', () => {
         cy.window().then(window => {
-            window.swup.loadPage({ url: '/page2' });
+            window.swup.loadPage({ url: '/page2/' });
             cy.shouldBeAtPage('/page2/');
             cy.shouldHaveH1('Page 2');
         });
     });
+
+    it('should update the body class', () => {
+        cy.window().then(window => {
+            window.swup.loadPage({ url: '/page2/' });
+            cy.get('body').should('have.class', 'body2');
+            cy.get('body').should('not.have.class', 'body1');
+        });
+    });
```

Comment:
This seems like plugin functionality being tested. I think we could use more of that to ensure the plugins are working correctly after updating Swup core itself. Should we start separating those tests into different places?

Reply:
Good point. Seeing how we're already testing other plugin functionality here (scroll plugin), I guess we can leave this in for now and tackle the plugin tests in a future PR. Moving the tests to the plugin repos could get a bit messy unless we create a new support repo `@swup/cypress-config` or something. Monorepo could be a solution, too. I'll start a discussion in Slack to gather ideas.